powqutyd/files/include/mqtt.h

```diff
@@ -10,8 +10,8 @@
 
 #include "PQ_App.h"
 
-#define MAX_MQTT_MSG_LEN		400
-#define mqtt_port 1883
+#define MAX_MQTT_MSG_LEN	1024
+#define mqtt_port		1883
```

Comment:
Should'nt we have this MARCO "mqtt_port" in upper case : "MQTT_PORT"

Reply:
I did not implement that macro and didn't want to change it, but can do so.